pkg/controller/core.oam.dev/v1alpha2/application/apply.go

```diff
@@ -51,33 +47,13 @@ import (
 	oamutil "github.com/oam-dev/kubevela/pkg/oam/util"
 )
 
-func errorCondition(tpy string, err error) runtimev1alpha1.Condition {
-	return runtimev1alpha1.Condition{
-		Type:               runtimev1alpha1.ConditionType(tpy),
-		Status:             corev1.ConditionFalse,
-		LastTransitionTime: metav1.NewTime(time.Now()),
-		Reason:             runtimev1alpha1.ReasonReconcileError,
-		Message:            err.Error(),
-	}
-}
-
-func readyCondition(tpy string) runtimev1alpha1.Condition {
-	return runtimev1alpha1.Condition{
-		Type:               runtimev1alpha1.ConditionType(tpy),
-		Status:             corev1.ConditionTrue,
-		Reason:             runtimev1alpha1.ReasonAvailable,
-		LastTransitionTime: metav1.NewTime(time.Now()),
-	}
-}
-
 type appHandler struct {
-	r                    *Reconciler
-	app                  *v1beta1.Application
-	appfile              *appfile.Appfile
-	previousRevisionName string
-	isNewRevision        bool
-	revisionHash         string
-	autodetect           bool
```

Comment:
why autodetect and appfile deleted?

Reply:
appfile is only used once as a parameter passed to finalizeAppRevision, it's not necessary make it as a field in appHandler.
`autodetect` is also not necessary to make it as a re-usable field, the logic is very easy. Before, we make it redundant.